Automattic/wp-calypso
# Site monitoring: Enable loading skeleton when changing tab from metrics to log

- Closes https://github.com/Automattic/dotcom-forge/issues/3832

## Proposed Changes

* Add skeleton loading to tables

## Testing Instructions



* On an atomic site, access `/site-monitoring/{siteSlug}`
* Click on `PHP Logs` or `Web Server Logs`
* Observe the loading skeleton for a few seconds

### Screencast


https://github.com/Automattic/wp-calypso/assets/779993/5792333f-74e1-4971-936e-77afeb48aab3


## Pre-merge Checklist
